CONTRIBUTING.md

```diff
@@ -111,7 +111,7 @@ make sure that the newly written code is well integrated into the existing packa
 module hierarchy.
 In particular, public functionality should be imported into the appropriate high-level
 namespaces for easier user import. For an example, see our
-[parameter namespace](baybe/parameters/__init__.py).
+[parameter namespace](https://github.com/emdgroup/BayBE/blob/main/baybe/parameters/__init__.py).
```

Comment:
is this now the gerneal recommended and best way to include links into the .md files? Asking because needs to be changed in a lot of README places too (not necessarily this PR)